hw/ip/lc_ctrl/doc/_index.md

```diff
@@ -495,9 +624,9 @@ Hence the following programming sequence applies to both SW running on the devic
 
 2. Read the {{< regref "LC_STATE" >}} and {{< regref "LC_TRANSITION_CNT" >}} registers to determine which life cycle state the device currently is in, and how many transition attempts are still available.
 
-3. Claim exclusive access to the transition interface by writing 1 to the {{< regref "CLAIM_TRANSITION_IF" >}} register, and reading it back. If the value read back equals to 1, the hardware mutex has successfully been claimed and SW can proceed to step 4. If the value read back equals to 0, the mutex has already been claimed by the other interface (either CSR or TAP), and SW should try claiming the mutex again.
+3. Claim exclusive access to the transition interface by writing 0xA5 to the {{< regref "CLAIM_TRANSITION_IF" >}} register, and reading it back. If the value read back equals to 0xA5, the hardware mutex has successfully been claimed and SW can proceed to step 4. If the value read back equals to 0, the mutex has already been claimed by the other interface (either CSR or TAP), and SW should try claiming the mutex again.
 
-4. Write the desired target state to {{< regref "TRANSITION_TARGET" >}}. If the transition is conditional, the corresponding token should be written to {{< regref "TRANSITION_TOKEN_0" >}} as well. An optional, but recommended step is to read back and verify the values written to these registers before proceeding with step 5.
+4. Write the desired target state to {{< regref "TRANSITION_TARGET" >}}. For conditional transitions, the corresponding token has to be written to {{< regref "TRANSITION_TOKEN_0" >}}. For all unconditional transitions, the token registers have to be set to zero. An optional, but recommended step is to read back and verify the values written to these registers before proceeding with step 5.
```

Comment:
For this line: the corresponding token has to be written to {{< regref "TRANSITION_TOKEN_0" >}}
We are not just writing to TRANSITION_TOKEN_0 right? We will use TRANSITION_TOKEN_0, TRANSITION_TOKEN_1, and TRANSITION_TOKEN_2. Maybe just use TRANSITION_TOKEN_*?

Reply:
That is true, although I found it a bit clunky to reference all registers, since they all are needed to store the full 128bit value... do you think it would be cleared to reference all registers of the multireg?

Reply:
Yeah i agree mentioning them all is a bit clunky. I was thinking about using `*` but it won't have a register link.  Yeah I am fine with what it is then :)

Reply:
OK sounds good (I know, using the * would indicate that better, but loosing the link is also not a good option, I think...)